packages/monaco/src/split-editor.tsx

```diff
@@ -62,6 +62,7 @@ export default function SplitEditor({
   monaco,
 }: SplitEditorProps) {
   const { settings, updateSettings } = useEditorSettingsStore();
+  const { subscribe } = useResetEditor();
```

Comment:
Just a personal preference but this is a similar pattern used in other places where `useResetEditor` is used.